### PR TITLE
Use multicall for balances

### PR DESCRIPTION
**Pull request by FiboApe**

### CODE REVIEW

File: src/common/token-picker/index.tsx [hunk 1]
```diff
@@ -166,12 +168,19 @@ const useListItemStyles = makeStyles(({ palette }) => ({
...
   },
 }));
-const Row = ({ index, style, data: { onClick, tokenList, tokenKeys, yieldOptions } }: RowProps) => {
+const Row = ({ index, style, data: { onClick, tokenList, tokenKeys, yieldOptions, tokenBalances } }: RowProps) => {
   const classes = useListItemStyles();
   const token = tokenList[tokenKeys[index]];
 
-  const tokenBalance = useTokenBalance(token);
-  const [tokenValue] = useUsdPrice(token, tokenBalance);
+  // const tokenBalance = useTokenBalance(token);
```
Comment (feraway) [inline src/common/token-picker/index.tsx:175]: You can probably delete this comment

File: src/common/token-picker/index.tsx [hunk 2]
```diff
@@ -321,14 +331,36 @@ const TokenPicker = ({
     currentNetwork.chainId,
   ]);
 
+  const [tokenBalances, isLoadingTokenBalances] = useMulticallBalances(memoizedUnorderedTokenKeys);
```
Comment (0xged) [inline src/common/token-picker/index.tsx:334]: Nice

File: src/home/swap-container/components/swap/index.tsx [hunk 3]
```diff
@@ -794,7 +792,7 @@ const Swap = ({
         isFrom={selecting === from}
         onChange={(from && selecting.address === from.address) || selecting.address === 'from' ? setFrom : setTo}
         usedTokens={usedTokens}
-        ignoreValues={ignoreValues}
+        ignoreValues={[]}
```
Comment (0xged) [inline src/home/swap-container/components/swap/index.tsx:795]: Noice

**Review by 0xged — APPROVED** [review]
Amazing work fibo 👑

**Review by feraway — APPROVED** [review]

**Pull request merged**